pkg/tsdb/elasticsearch/client/client.go

```diff
@@ -51,7 +51,7 @@ type ConfiguredFields struct {
 // Client represents a client which can interact with elasticsearch api
 type Client interface {
 	GetConfiguredFields() ConfiguredFields
-	ExecuteMultisearch(r *MultiSearchRequest) (*MultiSearchResponse, error)
+	ExecuteMultisearch(r *MultiSearchRequest) (*MultiSearchResponse, PluginError)
```

Comment:
you shouldn't have to do this.  the middleware will return PluginError which implements the error interface.